src/test/java/io/cdap/plugin/gcp/bigquery/BigQueryRecordToJsonTest.java

```diff
@@ -25,8 +25,12 @@
 import io.cdap.plugin.gcp.bigquery.sink.BigQueryRecordToJson;
 import org.junit.Assert;
 import org.junit.Test;
+import py4j.Base64;
```

Comment:
this is a weird one, did you mean to use java.util.Base64?

Reply:
Yes, fixed it.